feat: added alternative plugins section (#4469)
platformsh/platformsh-docs: sites/platform/src/add-services/opensearch.md

| `repository-s3`         | Support for using S3 as a repository for Snapshot/Restore                                 | *   | * |
| `transport-nio`         | Support for NIO transport                                                                 | *   | * |

### Alternative plugins

> **These plugins are currently available for `opensearch 2.0` only.** The names below show exactly how they should be added to your configuration. For example, to use the `alerting` plugin, specify it in your `services.yaml` as so:

Author: ```suggestion
**These plugins are currently available for open search `2.0` only.** The names below show exactly how they should be added to your configuration. For example, to use the `alerting` plugin, specify it in your `services.yaml` as so:
```

{{< note title="Different names used" theme="warning" >}}

It should be noted that **these names may differ to what they are commonly known as** in the [opensearch docs](https://opensearch.org/docs/latest/). To help your understanding of what each plugin does, please click each plugin name for a link to the corresponding Github repository. 

{{< /note >}}

| Plugin                                                                                                 | Description  | 1     | 2 |
---------------------------------------------------------------------------------------------------------|--------------|-------|---|
| [`alerting`](https://github.com/opensearch-project)| Monitor data and send alert notifications automatically                           | N/A   | * |
|[`opensearch-anomaly-detection`](https://github.com/opensearch-project/anomaly-detection)| Detect anomalies as your log data and monitor data in near real time                             | N/A   | * |
|[`asynchronous-search`](https://github.com/opensearch-project/asynchronous-search)|Run search queries in the background and retrieve results as they become available                                    | N/A   | * |

> |[`opensearch-cross-cluster-replication`](https://github.com/opensearch-project/cross-cluster-replication)|Replicate data across two opensearch clusters             | N/A   | * |

Author: ```suggestion
|[`opensearch-cross-cluster-replication`](https://github.com/opensearch-project/cross-cluster-replication)|Replicate data across two OpenSearch clusters             | N/A   | * |
```

|[`notifications`](https://github.com/opensearch-project/notifications)| A central location for all of your notifications from OpenSearch plugins                         | N/A   | * |
|[`opensearch-reports-scheduler`](https://github.com/opensearch-project/reporting)|Export and share reports from OpenSearch Dashboards dashboards, saved search, alerts and visualizations                                     | N/A   | * |
|[`geospatial`](https://github.com/opensearch-project/geospatial)|An OpenSearch plugin that contains geospatial specific features                                                     | N/A   | * |
|[`opensearch-index-management`](https://github.com/opensearch-project/index-management)|A suite of features to monitor and manage indexes                               | N/A   | * |
|[`opensearch-job-scheduler`](https://github.com/opensearch-project/job-scheduler)|Schedule periodical jobs running within OpenSearch nodes                                     | N/A   | * |
|[`opensearch-knn`](https://github.com/opensearch-project/k-NN)|Easily run the nearest neighbor search on billions of documents                                                        | N/A   | * |

> |[`opensearch-ml-plugin`](https://github.com/opensearch-project/ml-commons/)|Leverage existing opensource machine learning algorithms                                          | N/A   | * |

Author: ```suggestion
|[`opensearch-ml-plugin`](https://github.com/opensearch-project/ml-commons/)|Leverage existing Open source machine learning algorithms                                          | N/A   | * |
```
